pkgs/development/web/postman/linux.nix

```diff
@@ -0,0 +1,99 @@
+{ lib, stdenv, fetchurl, makeDesktopItem, wrapGAppsHook
+, atk, at-spi2-atk, at-spi2-core, alsa-lib, cairo, cups, dbus, expat, gdk-pixbuf, glib, gtk3
+, freetype, fontconfig, nss, nspr, pango, udev, libuuid, libX11, libxcb, libXi
+, libXcursor, libXdamage, libXrandr, libXcomposite, libXext, libXfixes
+, libXrender, libXtst, libXScrnSaver, libxkbcommon, libdrm, mesa, xorg, pname
+, version, meta
+}:
+
+stdenv.mkDerivation rec {
+  inherit pname version meta;
+
+  src = fetchurl {
+    url = "https://dl.pstmn.io/download/version/${version}/linux64";
+    sha256 = "sha256-pA3gT4xoIWhajY03JzVgHK5KyTx1uH6gyasuLTdt6cM=";
+    name = "${pname}.tar.gz";
+  };
+
+  dontBuild = true; # nothing to build
```

Comment:
```suggestion
```
If it does not break anything we shouldn't disable it.

Reply:
Jep, I did not have a look at the linux derivation because I didn't want to change/improve it in this pr. I'll add it.